src/constants/ssm-parameter-paths.ts

```diff
@@ -38,4 +39,9 @@ export const SSM_PARAMETER_PATHS: Record<string, SsmParameterPath> = {
     path: `${VPC_SSM_PARAMETER_PREFIX}/primary/subnets/public`,
     description: "A list of public subnets",
   },
+  FastlyCustomerId: {
+    path: "/account/external/fastly/customer.id",
+    description: "SSM parameter containing the Fastly Customer ID",
```

Comment:
Is this unique per Fastly service? And/or is it worth adding how one can obtain this value?

Reply:
It's a global value, I'll add a link to finding it here